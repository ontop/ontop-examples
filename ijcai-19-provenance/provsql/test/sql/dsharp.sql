\set ECHO none
SET search_path TO public, provsql;

CREATE TABLE dsharp_result AS
SELECT city, probability_evaluate(provenance(),'p','compilation','dsharp') AS prob
FROM (
  SELECT DISTINCT city
  FROM personnel
EXCEPT 
  SELECT p1.city
  FROM personnel p1,personnel p2
  WHERE p1.id<p2.id AND p1.city=p2.city
  GROUP BY p1.city
) t
ORDER BY city;

SELECT remove_provenance('dsharp_result');

SELECT * FROM dsharp_result;
DROP TABLE dsharp_result;
