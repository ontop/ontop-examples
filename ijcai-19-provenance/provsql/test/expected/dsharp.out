\set ECHO none
 remove_provenance 
-------------------
 
(1 row)

   city   | prob 
----------+------
 Berlin   | 0.54
 New York | 0.26
 Paris    | 0.41
(3 rows)

